Src/AutoFixture/Kernel/NoSpecimen.cs

```diff
@@ -12,12 +12,17 @@ namespace AutoFixture.Kernel
     /// <see langword="null"/> can be used as a proper return value.
     /// </para>
     /// </remarks>
-    public class NoSpecimen : IEquatable<NoSpecimen>
+    public sealed class NoSpecimen : IEquatable<NoSpecimen>
     {
+        /// <summary>
+        /// The singleton instance of <see cref="NoSpecimen"/>.
+        /// </summary>
+        public static readonly NoSpecimen Instance = new NoSpecimen();
```

Comment:
It wouldn't make a big difference for this particular singleton implementation but I would prefer a get-only auto-initialized property.